spec/models/site_status_spec.rb

```diff
@@ -18,126 +18,4 @@
     it { should belong_to(:site) }
     it { should belong_to(:course) }
   end
-
-  describe 'findable?' do
```

Comment:
As discussed IRL, can we find a way to include these tests, but maybe tidy them up to make them more readable? I'm concerned that even though these are technically being tested elsewhere, it's going to be hard to rationalise the test coverage for this model in the future when we don't remember that these methods are being tested elsewhere.

An idea for a custom matcher:

```
RSpec::Matchers.define :have_site_status do |*expected|
  match do |actual|
    if :findable.in? expected
      expect(SiteStatus.findable).to include actual
    end
  end
end
```

And using it:

```
specify do
   expect(subject).to have_site_status(:findable).and_not(:open_for_applications)
end
```